kubectl

```diff
@@ -19,18 +19,19 @@ if [ ! -f "$DEFAULT_CLIENT" ]; then
 	downloadClient "$DEFAULT_VERSION"
 fi
 
-if [ "$1" == "config" ]; then
-	$DEFAULT_CLIENT "$@"
-	exit $?
+VERSION_OUTPUT=$($DEFAULT_CLIENT version -o json 2>&1)
+if grep -q "Unable to connect to the server" <<<"$VERSION_OUTPUT"; then
```

Comment:
Depending on error message strings is fragile. Better to check the `exit code != 0`